### PR TITLE
Some cleanups

### PR DESCRIPTION
**Pull request by marcelm**
I’m trying to understand the code a bit better. While doing so, I found a few things that should improve readability. See individual commit comments. I can split this up into multiple PRs if desired.

### CODE REVIEW

File: main.nf [hunk 1]
```diff
@@ -312,28 +325,26 @@ process RealignerTargetCreator {
   """
 }
 
-verbose ? intervals = intervals.view {"Intervals to phase: $it"} : ''
+if (verbose) intervals = intervals.view {"Intervals to phase: $it"}
 
 bamsAndIntervals = duplicatesRealign
   .phase(intervals)
-  .map{duplicatesRealign, intervals ->
+  .map{dr, i ->
```
Comment (MaxUlysse) [inline main.nf]: I agree that all your changes go towards an improved readability, except for this one.
Does it really worth it to replacing `duplicatesRealign, intervals` by `dr, i` ?

File: main.nf [hunk 2]
```diff
@@ -312,28 +325,26 @@ process RealignerTargetCreator {
...
-      duplicatesRealign[0],
-      duplicatesRealign[1],
-      duplicatesRealign[2],
-      duplicatesRealign[3],
-      duplicatesRealign[4],
-      intervals[2]
+      dr[0],
+      dr[1],
+      dr[2],
+      i[2]
```
Comment (MaxUlysse) [inline main.nf]: I think the Travis error is caused by this `i[2]`, I'm guessing it should be `i[1]`

Comment (marcelm) [inline main.nf]: Ah, good, thanks! I don’t run the full tests locally and rely on Travis instead ;-)

**Comment by marcelm** [comment]
I’ve added one more tiny commit to this PR. Feel free to have a look. The tests also pass now.

**Pull request merged**